redfish-core/include/error_messages.hpp

```diff
@@ -417,6 +417,25 @@ nlohmann::json propertyValueIncorrect(const std::string& arg1,
 void propertyValueIncorrect(crow::Response& res, const std::string& arg1,
                             const std::string& arg2);
 
+/**
```

Comment:
Please let me know whether I need to push this commit upstream even without using it in the bmcweb?

Reply:
Yes, should be able too

Reply:
OK, Gerrit patch for the same commit: https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/52159